inference-engine/src/mkldnn_plugin/nodes/mkldnn_reduce_node.cpp

```diff
@@ -1807,7 +2146,7 @@ void MKLDNNReduceNode::reduce_BLK_concern_padding(const uint8_t *in_ptr, uint8_t
                             size_t oh = ReduceH ? 0 : ih; GET_PTR_NCDH_BLK;
                             parallel_for(IW, [&](size_t iw) {
                                 size_t ow = iw; GET_PTR_NCDHW_BLK;
-                                reduce_kernel_process(in_ptr_ncdhw, out_ptr_ncdhw, blk_size);
+                                reduce_kernel_process(in_ptr_ncdhw, out_ptr_ncdhw, blk_size, 2);
```

Comment:
M.b. it makes sense to introduce enum for `work_batch` because number `2` looks like a magic number.

Reply:
Fixed. I've roll-backed this revision. Thanks Maksim!